data/scripts/actions/others/golden_outfit_memorial.lua

```diff
@@ -0,0 +1,25 @@
+local goldenOutfitMemorial = Action()
+
+function goldenOutfitMemorial.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	local resultId = db.storeQuery("SELECT `player_id` FROM `player_storage` WHERE `key` = " .. PlayerStorageKeys.goldenOutfit .. " AND `value` >= 3;")
```

Comment:
`SELECT `player_id`, `name` FROM `player_storage` INNER JOIN `players` ON `name` = `name` WHERE `key` = " .. PlayerStorageKeys.goldenOutfit .. " AND `value` >= 3;`

Reply:
I haven't been successful in applying the changes yet.

Reply:
In a way it works partially, but all the chars of the account are showing instead of just showing the one that has the storage.

Reply:
wrong query sorry.
```db.storeQuery("SELECT `name` FROM `player_storage` INNER JOIN `players` as `p` ON `p`.`id` = `player_id` WHERE `key` = " .. PlayerStorageKeys.goldenOutfit .. " AND `value` >= 3;")
```

Reply:
works fine, thanks!!